### PR TITLE
refactor: v3

### PR DESCRIPTION
**Pull request by Louis-Deez**
## Summary of changes
This is a new website design

### CODE REVIEW

File: src/components/home/A320Header.tsx [hunk 1]
```diff
@@ -20,19 +20,21 @@ export const A320Header = () => {
...
                 loop
             />
-            <Container className="relative max-w-6xl min-h-screen">
+            <Container className="relative min-h-screen">
                 <div className="absolute bottom-0 left-0 h-2/3 px-page">
-                    <h1 className="text-8xl font-bold">
-                        <span className="text-teal">A32</span>
-                        <span className="text-blue">N</span>
-                        X
+                    <h3 className="text-3xl font-semibold text-cyan-dark">
```
Comment (NathanInnes) [inline src/components/home/A320Header.tsx:25]:
```suggestion
                    <h3 className="text-3xl font-semibold text-primary">
```

We should look into named colours. See [here](https://tailwindcss.com/docs/customizing-colors#naming-your-colors)

**Comment by NathanInnes** [comment]
Converting to draft in the mean time whilst this revamp is underway.

**Comment by Benjozork** [comment]
We need it as active for vercel

**Comment by NathanInnes** [comment]
I opened #466, I do not intend to step on toes, I would like to improve everything on this REPO to make it a much cleaner place to work. I will integrate many changes from this PR as well.

**Pull request closed without merging**